alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md

```diff
@@ -0,0 +1,105 @@
+# PrometheusScrapeBodySizeLimitHit
+
+## Meaning
+
+The alert `PrometheusScrapeBodySizeLimitHit` is triggered when at least one of
+Prometheus' scrape targets replies with a response body larger than `body_size_limit`.
+
+## Impact
+
+The metrics coming from the endpoint will not be scraped each time when
+the response body size exceeds the body size limit.
+
+## Diagnosis
+
+First we need check the value of `body_size_limit` in the Prometheus configuration
+file, which is saved in the secret `prometheus-k8s` in the namespace `openshift-monitoring`.
+Search for the field `body_size_limit`.
+
+```bash
+kubectl get secret -n openshift-monitoring prometheus-k8s -o yaml | grep prometheus.yaml.gz: | sed -e 's/^[ ]*prometheus.yaml.gz: *//' | base64 -d | gunzip -c
+```
+
+If the value is too small, we need to set it to a larger number. Please refer to
+[Mitigation](#mitigation) on how to change the value.
+
+If the value is correct, we need to find out which target is responding with a body
+too large for scraping. To do this, we can query the Prometheus API endpoint `/api/v1/targets`,
+in `data.activeTargets` search for those who have `health` field different than `up`,
+checking for the `lastError` field for confirmation:
```

Comment:
The targets page (OCP console) should report the faulty endpoints too.

Reply:
will add a phrase telling user to refer to the target page in UI first. If more information needed, we can also using the API endpoint for more details.